src/Parser/Warlock/Demonology/Modules/Features/Felstorm.js

```diff
@@ -27,6 +27,15 @@ class Felstorm extends Analyzer {
     }
   }
 
+  get suggestionThresholds(){
+    const maxCasts = Math.ceil(calculateMaxCasts(FELSTORM_COOLDOWN, this.owner.fightDuration));
```

Comment:
I never thought about pet abilities before! Have you tried putting this in the Abilities class too? Might need a bit too much work to get it to properly track the CD in the timeline, so might not be worth the effort for now.

Reply:
It's less a pet ability and more a CD you get your pet to use either automatically or manually. It can be added to Abilities fairly easily just like any other CD.